src/Application/Rokalo.Application/User/Commands/RegisterUserCommand.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace Rokalo.Application.User.Commands
+{
+    using FluentValidation;
+    using MediatR;
+    using Rokalo.Application.Contracts;
+    using Rokalo.Application.Helpers;
+    using Rokalo.Application.Services;
+    using Rokalo.Domain;
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public record RegisterUserCommand(
+        string Email,
+        string Password,
+        string FirstName,
+        string LastName,
+        string PhoneNumber,
+        string MobileNumber,
+        string Address,
+        int CityId) : IRequest;
+
+    internal sealed class RegisterUserCommandValidator : AbstractValidator<RegisterUserCommand>
+    {
+        public RegisterUserCommandValidator()
+        {
+            RuleFor(u => u.Password).Password();
+            RuleFor(u => u.FirstName).Length(0, 50);
+            RuleFor(u => u.LastName).Length(0, 50);
+            RuleFor(u => u.PhoneNumber).Length(0, 20);
+            RuleFor(u => u.MobileNumber).Length(0, 20);
+            RuleFor(u => u.Address).Length(0, 100);
+            RuleFor(u => u.CityId).NotNull(); // TODO add must be valid city id, where is that city id?
```

Comment:
We can create table "cities" and feed some information in that table.
Then use those id for CityId